PublicMapping/districtbuilder
# Select geounits at any geolevel

## Overview

Select geounits at any geolevel

### Demo

![db_geolevels_selection](https://user-images.githubusercontent.com/2926237/88081327-38db3880-cb4e-11ea-8929-bfebaaec786a.gif)

### Notes

Does not include making demographics work for each level (there are several TODOs that refer to #202 where that work will happen)

Also does not include ensuring zoom levels are appropriate for selected geolevel (see #201). You may notice that if you try to select blocks when you're too far zoomed out to see them it looks broken but it's not -- you just can't see them!

Includes a change to `process-geojson` script to add the index for the geounit itself and not just the parents

## Testing Instructions
* Re-process your GeoJSON or use mine for DE (s3://global-districtbuilder-dev-us-east-1/regions/US/DE/2020-07-21T00:58:57.935Z/)
* Using the default selection tool, assign geounits at different levels and make it work appropriately
* Test the rectangle selection tool to make sure that still works too 

Closes #200
